app/models/assignment.rb

```diff
@@ -127,7 +128,7 @@ def run_tests!(params)
   end
 
   def to_resource_h
-    as_json(except: [:exercise_id, :submission_id, :organization_id, :id, :submitter_id, :solution, :created_at, :updated_at, :submission_status],
+    as_json(except: [:exercise_id, :submission_id, :organization_id, :id, :submitter_id, :solution, :created_at, :updated_at, :submission_status, :submitted_at],
```

Comment:
Aren't we going to expose the `submitted_at` field? Why?

Reply:
Look at my comment https://github.com/mumuki/mumuki-domain/pull/50#issuecomment-528508029